src/modules/material-requests/controllers/material-requests.controller.ts

```diff
@@ -36,4 +37,13 @@ export class MaterialRequestsController {
 
 		return visits;
 	}
+
+	@Get(':id')
+	@RequireAnyPermissions(
+		Permission.READ_ALL_MATERIAL_REQUESTS,
```

Comment:
Rename to better match the naming convention for the other permissions
```suggestion
		Permission.VIEW_ANY_MATERIAL_REQUESTS,
```

Reply:
Adjusted